packages/strapi-parts/src/Combobox/Combobox.js

```diff
@@ -176,42 +189,85 @@ export const Combobox = ({
     });
   });
 
+  const handleClear = () => {
+    inputRef.current.focus();
+
+    if (onClear) {
+      onClear();
+    }
+
+    setInputValue('');
+  };
+
+  const handleCaretClick = () => {
+    inputRef.current.focus();
+    updateMenuState(true);
+  };
+
   return (
     <Field hint={hint} error={error} id={generatedId}>
       <Stack size={label || hint || error ? 1 : 0}>
         {label && <FieldLabel id={labelId}>{label}</FieldLabel>}
         <MainRow ref={containerRef} $disabled={disabled} hasError={error}>
-          <Input
-            id={generatedId}
-            aria-activedescendant={activeId}
-            aria-autocomplete="list"
-            aria-controls={`${generatedId}-listbox`}
-            aria-expanded={`${open}`}
-            aria-haspopup="listbox"
-            aria-describedby={label ? labelId : undefined}
-            aria-disabled={disabled}
-            readOnly={disabled}
-            ref={inputRef}
-            role="combobox"
-            type="text"
-            value={inputValue}
-            onBlur={disabled ? undefined : onInputBlur}
-            onClick={disabled ? undefined : () => updateMenuState(true)}
-            onInput={disabled ? undefined : onInput}
-            onKeyDown={disabled ? undefined : onInputKeyDown}
-            placeholder={placeholder}
-            {...props}
-          />
+          {disabled ? (
```

Comment:
Not sure why this big ternary is necessary? What are the very big differences between the "else" part of it and the "if" part of it?